src/Sylius/Bundle/ApiBundle/DependencyInjection/Compiler/SyliusLegacyAliasesPass.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class SyliusLegacyAliasesPass implements CompilerPassInterface
+{
+    public const NEW_PREFIXES_TO_LEGACY_MAP = [
```

Comment:
```suggestion
    public const PLUGIN_TO_SYLIUS_MAP = [
```
or
```suggestion
    public const PLUGIN_TO_SYLIUS_NAMESPACES_MAP = [
```
wdyt? I'm also in favor of the map inversion